### PR TITLE
sf-9045 | Expose get/set of logging status as commands

### PR DESCRIPTION
**Pull request by MicheleBuselli**

### CODE REVIEW

File: src/commands/diagnostics/logging/update.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+export = {
+	command: 'update',
+	description: 'Update logging configuration',
+	builder: yargs => {
+		yargs.options({
+			'level': {
+				alias: 'l',
+				desc: 'Set the level of detail for application logs',
+				type: 'string'
+			},
```
Comment (Alex-Murtha) [inline src/commands/diagnostics/logging/update.ts:14]: We should take advantage of yargs `choices` feature for log level: https://yargs.js.org/docs/#api-reference-choiceskey-choices

**Review by Alex-Murtha — CHANGES_REQUESTED** [review]

File: src/commands/diagnostics/logging/util.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { DiagnosticLoggingStatus, LoggingLevel } from '../../../sdk';
+import { OutputFormatter } from '../../../util/formatter';
+
+export const getLevels = (enumObject: typeof LoggingLevel) : string[] => {
+	let levels = [];
+	for (let key in enumObject) {
+		if ({}.hasOwnProperty.call(enumObject, key)) {
```
Comment (MicheleBuselli) [inline src/commands/diagnostics/logging/util.ts:7]: This is weird but it gets rid of an eslint warning

Comment (Alex-Murtha) [reply src/commands/diagnostics/logging/util.ts:7]: Try using this:
```
export const getLogLevels = () : string[] => {
  return Object.keys(LoggingLevel).map(key => LoggingLevel[key]); // possibly use Object.values(LoggingLevel)?
}
```
https://stackoverflow.com/questions/39372804/how-can-i-loop-through-enum-values-for-display-in-radio-buttons

Comment (Alex-Murtha) [reply src/commands/diagnostics/logging/util.ts:7]: Might be worth moving this to models as well so the UI can use this function and not have duplicate logic

**Review by Alex-Murtha — APPROVED** [review]

**Pull request merged**